app/serializers/api/v1/group_serializer.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+class Api::V1::GroupSerializer
+  include FastJsonapi::ObjectSerializer
+
+  attributes :member_count do |group|
+    group.group_members.count
```

Comment:
can we use counter cache here to avoid N+1 query?

Reply:
That would be great addition, thanks for this suggestion!!